lib/utils/text.rb

```diff
@@ -11,5 +11,9 @@ def sanitze_for_chat(text)
       # Remove bash colorings
       text.gsub(/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]/, '')
     end
-  end
+
+    # camel case puppet classes
+    def class_camel(text)
+      text.split('::').map(&capitalize).join('::')
```

Comment:
The block shortcut (`&`) for the `#map()` method expects a symbol, meaning `capitalize` should be `:capitalize`. Otherwise this looks reasonable.